app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if command.startswith("cp"):
+        temp = command.split()
+        old_file, new_file = temp[1], temp[2]
+        if temp[1] != temp[2]:
+            with open(old_file, "r") as file_in, open(
+                new_file, "w"
+            ) as file_out:
```

Comment:
You can increase readability in this case with brackets:

`(with open(old_file, "r") as file_in,`
`with open(new_file, "w") as file_out):`

Reply:
my python 3.7 doesn`t support it 
![image](https://user-images.githubusercontent.com/123904633/220343603-cc32797c-ea2d-4d23-80ed-85864701de44.png)

Reply:
((